src/manager/index.js

```diff
@@ -83,17 +84,32 @@ const CacheAPI = {
     // FIXME these will be made mandatory
     deviceModel?: DeviceModelId,
     deviceInfo?: DeviceInfo
-  ): string => {
+  ): string[] => {
```

Comment:
strictly speaking a change of interface be a MAJOR change, not a PATCH bump
and i see you only made a patch semver change mmh